app/models/git/website_copy.rb

```diff
@@ -5,7 +5,7 @@ class WebsiteCopy
     DEFAULT_REPO_URL = "https://github.com/exercism/website-copy".freeze
 
     def initialize(repo_url: DEFAULT_REPO_URL)
-      @repo = Repository.new(nil, repo_url: repo_url)
+      @repo = Repository.new(repo_url: repo_url)
```

Comment:
The other `git` classes allow a `repo` parameter to be passed for testing purposes. We don't yet need it, but I wanted to mention the inconsistency

Reply:
The `repo` param was actually intended as a perfomance optimisation, so that we could pass the v3 repo (and maybe now a track repo) around different exercises, rather than creating a new one for each. I'm unsure whether there really is a performance benefit - my hope was that the actual repo was just held in memory once. 

The reason I didn't do that here is that this repo is standalone, in the sense that nothing else shares the underlining repo with this class.

Reply:
Okay, makes sense